task3/Makefile

```
@@ -0,0 +1,104 @@
+CС=gcc
+CFLAGS= -Wall -Werror -Wextra
+
+SOURCES=lb1task3.c # Тут вы перечислятете файлы с вашими лабами по 1 файлу на задание
+TEST_TEMPLATES_DIR=/mnt/c/Users/Вероника/Desktop/lb1/test_templates
+EXECUTABLES=$(SOURCES:.c=.a)
+BASH=bash
+SUCCESS_CMD=$(BASH) $(TEST_TEMPLATES_DIR)/success_cmd.txt

```

Убрать везде *.txt
Чтобы тесты заработали надо в директории test_templates выполнить
```bash
sudo chmod +x *
```
```suggestion
SUCCESS_CMD=$(BASH) $(TEST_TEMPLATES_DIR)/success_cmd
...
```